frontend/src/app/contribute.cljs

```diff
@@ -177,7 +177,10 @@
                 :on-change #(on-how-to-fix-textarea-change (.-target %))}]]
 
    [:div {}
-    [:label {:class "form-label"} "Ready to submit the results?"]
+    [:label {:class "form-label"} "Ready to submit the results?"
```

Comment:
You are missing a space after the question mark

Reply:
Fixed